suite-native/accounts/src/selectors.ts

```diff
@@ -69,13 +36,22 @@ export const selectFilteredAccountsGroupedByNetwork = memoizeWithArgs(
             })),
             accountsWithFiatRatedTokens =>
                 filterAccountsByLabelAndNetworkNames(accountsWithFiatRatedTokens, filterValue),
-            A.groupBy(account => account.symbol),
-        );
+            groupAccountsByNetworkAccountType,
+        ) as GroupedAccounts;
     },
     // This selector is used only in one search component, so cache size equal to 1 is enough.
     { size: 1 },
 );
 
+export const selectNetworkAccountsGroupedByAccountType = memoizeWithArgs(
+    (state: AccountsRootState & DeviceRootState, networkSymbol: NetworkSymbol) =>
+        pipe(
+            selectDeviceAccountsByNetworkSymbol(state, networkSymbol),
+            groupAccountsByNetworkAccountType,
+        ) as GroupedAccounts,
+    { size: 3 }, // Average user does not have more than 3 assets synced.
```

Comment:
This should not be average but maximal expected value, otherwise for users with more than 3 assets it could cause rerenders with every change anywhere in state.

Reply:
fixed in f8abb496a6